torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12265,6 +12272,15 @@ def ref_pairwise_distance(input1, input2):
            # https://github.com/pytorch/pytorch/issues/66357
            check_batched_forward_grad=False,
            sample_inputs_func=sample_inputs_transpose_swapdims),
+    OpInfo('transpose_copy',
+           ref=lambda x, dim0, dim1: ref_transpose(x, dim0, dim1).copy(),
+           # transpose_copy does not support automatic differentiation for outputs with complex dtype
+           dtypes=all_types_and(torch.bool, torch.bfloat16, torch.half),
+           dtypesIfCUDA=all_types_and(torch.bool, torch.bfloat16, torch.half),
+           supports_out=False,
```

Comment:
It should support out. Are the tests failing?

Reply:
`transpose_copy` does not have `out` argument defined.

Reply:
`native_functions.yaml` have this defined:
```
- func: transpose_copy.out(Tensor self, int dim0, int dim1, *, Tensor(a!) out) -> Tensor(a!)
```

Reply:
Yes, but `out=` is still not functional:
```python
>>> import torch
>>> x = torch.tensor([[1, 2], [3, 4]])
>>> r = x.transpose_copy(0, 1)
>>> x.transpose_copy(0, 1, out=r)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: transpose_copy() got an unexpected keyword argument 'out'
```
not sure why..

Reply:
Ah, got it: the `out=` is available only for torch function, not for tensor methods.

Reply:
Removed `supports_out=False`.